src/components/DropdownSelect.js

```diff
@@ -9,69 +9,92 @@ import { Wrapper, WrapperRow, WrapperHorizontal } from './Wrapper';
 import { Icon } from './Icon';
 import { arrowDown, arrowUp } from '../icons';
 import { Label } from './Label';
+import { OrientationContext } from '../OrientationProvider';
 
-export const DropdownSelect = memo(
-  ({ data, setData, label, showSearch, searchInputStyle, renderSearch, searchPlaceholder }) => {
-    if (!data || !data.length) return null;
+export const DropdownSelect = ({
+  data,
+  setData,
+  label,
+  showSearch,
+  searchInputStyle,
+  renderSearch,
+  searchPlaceholder
+}) => {
+  const { orientation } = useContext(OrientationContext);
 
-    const [arrow, setArrow] = useState('down');
-    const selectedData = data.find((entry) => entry.selected);
-    const selectedIndex = data.findIndex((entry) => entry.selected);
-    const preselect = (index) => this[`dropdown${label}`].select(index);
+  const marginHorizontal =
+    orientation === 'portrait' ? normalize(14) : device.height * 0.15 + normalize(14);
```

Comment:
what is the calculation of `device.height * 0.15` based on here?

Reply:
ok i see, `0.15` is the 15% landscape wrapper padding

Reply:
i was wondering, why the dropdown modal wasn't full width on landscape.

changing to

```
// 0.15 = 15% landscape wrapper padding
const marginHorizontal = orientation === 'portrait' ? normalize(14) : device.width * 0.15;
```

results in

![Simulator Screen Shot - iPhone 11 Pro Max - 2021-03-02 at 11 52 24](https://user-images.githubusercontent.com/1942953/109638090-dfa89e00-7b4d-11eb-9fba-311867bbb6ff.png)

Reply:
and... you prefer that?
i am not quite sure what to make of this comment? xD

Reply:
hmmm...but i think that list has no landscape wrapper. we need to reduce the full width by safe area size. otherwise devices without safe areas have no full width here...which could be fine as well.

Reply:
with `const { left: safeAreaLeft, right: safeAreaRight } = useSafeAreaInsets();` (from `react-native-safe-area-context`) we can get the safe are insets:

```
"insets": Object {
  "bottom": 21,
  "left": 44,
  "right": 44,
  "top": 0,
}
```

Reply:
so we need to end up with

```
const marginHorizontal = orientation === 'portrait' ? normalize(14) : safeAreaLeft + normalize(14);
```

Reply:
![Simulator Screen Shot - iPhone 11 Pro Max - 2021-03-02 at 12 23 54](https://user-images.githubusercontent.com/1942953/109641818-711a0f00-7b52-11eb-9f16-aac34ecf19ea.png)

![IMG_2519B2B94384-1](https://user-images.githubusercontent.com/1942953/109641827-72e3d280-7b52-11eb-872a-443a7f90006a.jpeg)

Reply:
looks good to me on both devices as it is now